sdk/storage/azure_storage_blob/src/clients/blob_client.rs

```diff
@@ -17,6 +17,7 @@ use azure_core::{
 };
 use std::sync::Arc;
 
+// A client to interact with a specific Azure storage blob, although that blob may not yet exist.
```

Comment:
Triple slashes for a proper doc comment.
```suggestion
/// A client to interact with a specific Azure storage blob, although that blob may not yet exist.
```

Reply:
Good catch, bad copy and paste of a double slash for all the handwritten clients. Will fix in next commit, but pending your response wrt `lib.rs`

Reply:
I fixed this.